docs/sources/mobile-app/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Grafana OnCall mobile app
+aliases:
+  - /docs/oncall/latest/mobile-app/
+keywords:
+  - Mobile App
+  - oncall
+  - notification
+  - push notification
+weight: 1200
+---
+
+# Grafana OnCall Mobile App
+
+>**Note**: This application is currently in beta
+
+The Grafana OnCall mobile app allows teams to monitor and respond to critical system events from anywhere.
+On-call engineers can reimagine incident response with real-time alerts, on-call notifications, and a growing feature set.
```

Comment:
The reality is that these features are table stakes, and we're being handwavey here about the future.  Which is OK but calling this "reimagining" is a big stretch. If I was a customer or prospect, this wouldn't land well

Reply:
Roger that! Totally agree and it's a bit too market-ey anyways. It was just reused content from the app store description which is more market-ey, by nature. Will remove and reword this section to set more realistic expectations.